feature-libs/organization/user-registration/components/form/user-registration-form.service.ts

```diff
@@ -172,7 +172,9 @@ export class UserRegistrationFormService {
     if (
       this.authConfigService.getOAuthFlow() ===
         OAuthFlow.ResourceOwnerPasswordFlow ||
-      this.featureConfigService.isEnabled('authorizationCodeFlowByDefault')
+      this.featureConfigService.isEnabled(
```

Comment:
This change is related to the enhancements we made to support oAuth flows involving a redirect.  It is not related to the change in the default auth config.